e2e/e2e-test-setup.js

```diff
@@ -0,0 +1,26 @@
+import { execSync } from 'child_process'
+import * as path from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+const appDir = path.join(__dirname, 'test-app')
+
+execSync('cargo build --release', { stdio: 'inherit' })
+
+const isWindows = process.platform === 'win32'
+const tuonoBuildDir = path.join(
+  __dirname,
+  '..',
+  'target',
+  'release',
+  isWindows ? 'tuono.exe' : 'tuono',
+)
+
+execSync(`${tuonoBuildDir} new test-app --template tuono-tutorial`, {
+  cwd: __dirname,
+  stdio: 'inherit',
+})
+
+execSync('npm install', { cwd: appDir, stdio: 'inherit' })
```

Comment:
```suggestion
execSync('pnpm install', { cwd: appDir, stdio: 'inherit' })
```

Reply:
It was intentional since here we could use any package manager so I wanted to have a default use case. Do you think `pnpm` would bring more benefits in this case?

Reply:
Usually `pnpm` is faster than `npm` . 
I would consider using it because Windows workflows can be a real pain when it comes to execution time 😂.

Reply:
That makes sense, I also noticed Windows workflows are slow. Should be fixed now 👍